.github/workflows/playwright-actions.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Create front-end .env file
         run: |
           # Check if the secrets are set
-          if [ -z "$" ]; then
+          if [ -z "$ADMIN_USERNAME" ]; then
```

Comment:
? Seems obvious, but some mention in commit message would be nice :)

Reply:
thought it was obvious, but amended it to the commit message 🫡😅